### ISSUE TITLE
(BUG) server is not accepting new accounts

### ISSUE THREAD

**Issue by Yiyi-philosophy** [enchant97/note-mark#210]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is This A Security Issue?

- [X] This is not a security issue

### Describe The Problem

After building docker:
```docker
# file: docker-compose.yml
# image: 0.14.0
version: "3"
volumes:
  data:
services:
  note-mark:
    image: ghcr.io/enchant97/note-mark-aio
    restart: unless-stopped
    volumes:
      - data:/data
    environment:
      JWT_SECRET: "xxx"
      CORS_ORIGINS: "*"
      ALLOW_SIGNUP: "true"
    ports:
      - 58600:8000
  ```
  
 I visit `http://192.168.2.14:58080/login` and try to **create user**.
![image](https://github.com/user-attachments/assets/c7cfaefd-263e-4bfa-818e-ec4c9eeb4d6e)

It shows this:
`server is not accepting new accounts`

and docker logs:

```shell
note-mark-1  | {"time":"2024-09-25T08:58:39.820091241Z","id":"","remote_ip":"192.168.2.7","host":"192.168.2.14:58600","method":"POST","uri":"/api/users","user_agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:130.0) Gecko/20100101 Firefox/130.0","status":403,"error":"","latency":402901,"latency_human":"402.901µs","bytes_in":48,"bytes_out":0}
```

### Expected Behavior

I'm not sure what's happening here. I hope someone could point out what causes this problem.When trying to create a new user at http://192.168.2.14:58600/login, I expect to successfully register an account.

### Steps To Reproduce

use my `docker-compose.yml`
and `docker-compose up -d`

### Environment

Linux 22.04.3-Ubuntu x86

```docker
# file: docker-compose.yml
# image: 0.14.0
version: "3"
volumes:
  data:
services:
  note-mark:
    image: ghcr.io/enchant97/note-mark-aio
    restart: unless-stopped
    volumes:
      - data:/data
    environment:
      JWT_SECRET: "xxx"
      CORS_ORIGINS: "*"
      ALLOW_SIGNUP: "true"
    ports:
      - 58600:8000
  ```

### Extra Context

tried Edge and Firefox, both the same error.

**Comment by Yiyi-philosophy** [comment]
Update: This bug fixed when I set **docker-version to 0.13.0**.

**Comment by enchant97** [comment]
Thanks for reporting, will be fixed shortly!
